static/Gruntfile.js

```diff
@@ -12,34 +12,30 @@ module.exports = function (grunt) {
     watch: {
       all: {
         files: ['**', '!**/node_modules/**', `!${outputDir}/**`],
-        tasks: ['jambobuild', 'webpack',],
-        options: {
-          spawn: false,
-        },
+        tasks: ['jambobuild']
       },
     },
   });
 
   grunt.loadNpmTasks('grunt-webpack');
   grunt.loadNpmTasks('grunt-contrib-watch');
+  grunt.registerTask('jambobuild', 'Jambo build.', jambobuild);
+}
 
-  grunt.registerTask('jambobuild', 'Jambo build.',
-  function() {
-    // Force task into async mode and grab a handle to the "done" function.
-    var done = this.async();
-    // Run some sync stuff.
-    grunt.log.writeln('Processing task...');
-    // And some async stuff.
-    exec('npx jambo build', (error, stdout, stderr) => {
-      if (error) {
-        console.log(error.message);
-        done(false);
-        return;
-      }
-
-      stderr && console.error(stderr);
-      stdout && console.log(stdout);
-      done();
-    });
+/**
+ * Spawns a jambo and webpack build and prints sends all output to the console
+ */
+function jambobuild () {
```

Comment:
Since this does more than just the Jambo build, could we change the name? Maybe something like `buildSite` or something like that?

Reply:
sure

Reply:
I think it's okay to rename the task. I seriously doubt anyone is invoking it themselves and as you said, the CI scripts don't reference it.